api/tests/integration/core/test_user_rate_throttle.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from django.urls import reverse
+from pytest_lazyfixture import lazy_fixture
+from rest_framework import status
+
+
+@pytest.mark.parametrize(
+    "client",
+    [(lazy_fixture("admin_master_api_key_client")), (lazy_fixture("admin_client"))],
+)
+def test_user_throttle_can_throttle_admin_endpoints(
+    client, project, mocker, reset_cache
+):
+    # Given
+    mocker.patch("core.throttling.UserRateThrottle.get_rate", return_value="1/minute")
```

Comment:
Shouldn't we use the `settings` fixture here? Or is that painful because of the nested dictionary issues we had previously?

Reply:
Yep, correct!  Settings fixture does not work (correctly) for nested data structures